### ISSUE TITLE
Backend User management

### ISSUE THREAD

**Issue by benbieler** [SocialDocs/SocialDocs#2]
Backend User management:
- Build REST API
- Token Authentication

**Comment by Ma27** [comment]
am not that familar with sails but is there any kind of user management plugin??

**Comment by benbieler** [comment]
Yes, of corce, passport would be a good choise. I havent jet realy started to work on this issue, due to a lack of time.
Whould you, @Ma27 , agree with using passport for user authentication?

**Comment by Ma27** [comment]
I have to have a look at this, then I can confirm whether or not to use this...

Von meinem iPhone gesendet

**Comment by benbieler** [comment]
Well I actually think, that passport is the best way. Sails supports this really well, and passport is the '"norm" when it comes to NodeJs authentication...
